flytekit/plugins/__init__.py

```diff
@@ -15,6 +15,8 @@
 hmsclient = _lazy_loader.lazy_load_module("hmsclient")  # type: _lazy_loader._LazyLoadModule
 type(hmsclient).add_sub_module("genthrift.hive_metastore.ttypes")
 
+sagemaker_training = _lazy_loader.lazy_load_module("sagemaker-training")  # type: _lazy_loader._LazyLoadModule
```

Comment:
@kumare3 Maybe I misunderstood what Lazy Load is used for? `sagemaker_training` needs to be installed in the container if the users wants to do custom training on SM because it will put a `train` script in PATH. It's not really used in user code but it needs to be installed.

Reply:
cc @EngHabu ^